bin/get-individual-scoring-finesse.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import ailab.db.finesse as finesse
+import ailab.db as db
+import pandas as pd
+import dotenv
+
+# This is used to load the .env file
+dotenv.load_dotenv()
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+def create_individual_scoring(cursor):
+    """Create individual scoring view."""
+    with open("sql/2023-11-02-individual-scoring-view.sql", "r") as file:
+        sql_command = file.read()
+    cursor.execute(sql_command)
+
+if __name__ == "__main__":
+    """Get the individual scoring of each documents and write the results in a csv file."""
+    connection = db.connect_db()
+    with db.cursor(connection) as cursor:
+        create_individual_scoring(cursor)
+        res = finesse.get_individual_scoring(cursor)
+    with open('tests/output/individual_scoring.csv', 'r+', newline='') as file:
```

Comment:
r+? shouldn't it be w+ to create the file?

Reply:
newline=''? ok, I think we need to talk as based on this and your other comment I'm unsure you're respecting the spec of what the output should be.

Reply:
As discussed, I used multiple documentation.
Here is the one one r+ and w+ but we should use w+ to create the file every time so it doesn't create an error if the file doesn't exist.
https://www.freecodecamp.org/news/file-handling-in-python/

Here is the one I used first time to understand how to open a csv file and write in it :
https://docs.python.org/3/library/csv.html 

For me it was logic to use r+ as I created the file myself instead of doing it in the code.

With further investigation, turns out that I won't need to use open with to_csv based on this documentation : 
https://www.digitalocean.com/community/tutorials/pandas-to_csv-convert-dataframe-to-csv

I will apply changes to have a better code.